examples/batch/batch.go

```diff
@@ -85,20 +85,20 @@ func requestBatch(db PgxIface) (err error) {
 	// Iterate over a batch of queued queries
 	for _, query := range batch.QueuedQueries {
 
-		// Print SQL statement of a queued query
+		// Print SQL statement of the current query
 		fmt.Println(query.SQL)
 
-		// BatchResult.Query reads results from a queued query
+		// BatchResult.Query reads results from the current query
 		rows, err := br.Query()
```

Comment:
The first query is `INSERT`. It returns no rows. It's wrong to use `br.Query` for it